app/com/gu/aws/AwsS3Client.scala

```diff
@@ -13,9 +14,11 @@ import scala.util.{Failure, Success, Try}
 
 object AwsS3 extends StrictLogging {
 
-  lazy val client = AmazonS3Client.builder.withCredentials(CredentialsProvider).build()
+  lazy val client = AmazonS3Client.builder.withCredentials(CredentialsProvider).withRegion(Regions.EU_WEST_1).build()
```

Comment:
For my understanding - how did this work previously?

Reply:
The first time I ran the application locally I got this error.

<img width="1344" alt="Screenshot 2024-06-06 at 14 45 22" src="https://github.com/guardian/memsub-promotions/assets/39066365/e19784de-0ab6-4533-ae7f-9aae5139d2ec">

So, I added the region the same way it is present in the same repo [here](https://github.com/guardian/memsub-promotions/blob/2d75c29276f0d3e253171c8d7bca79d5f982ce35/app/com/gu/memsub/auth/common/S3PrivateKeyService.scala#L17).